simplejson/_speedups.c

```diff
@@ -277,6 +279,37 @@ moduleinit(void);
 
 #define MIN_EXPANSION 6
 
+static PyObject* JsonStrType;
+#if PY_MAJOR_VERSION < 3
+static PyObject* JsonUniType;
+#endif /* PY_MAJOR_VERSION < 3 */
+static int is_json_type(PyObject *pystr)
+{
+    if (JsonStrType == NULL) {
+        PyObject *encoder_module = PyImport_ImportModule("simplejson.encoder");
+        JsonStrType = PyObject_GetAttrString(encoder_module, "_JsonStr");
+        Py_DECREF(encoder_module);
+        if (JsonStrType == NULL)
+            return 0;
+    }
+    if (PyObject_isInstance(pystr, JsonStrType))
```

Comment:
I think you may not be testing the C extension, I'm fairly certain that the correct capitalization of this function is `PyObject_IsInstance`